training-service/src/app/content/subcategory/component/subcategory-add/subcategory-add.component.css

```diff
@@ -0,0 +1,11 @@
+.padding10 {
+  padding: 10px 0px;
+}
+
+.textarea-padding {
+  padding: 10px 0px;
+}
+
+.not-valid {
+  color: red;
```

Comment:
Please create a "color palette", a file with variables representing colors and use here a color variable instead of color directly.

Reply:
Done.